mp/src/game/shared/momentum/mom_gamerules.cpp

```diff
@@ -417,53 +413,14 @@ void CMomentumGameRules::RadiusDamage(const CTakeDamageInfo &info, const Vector
             flDistanceToEntity = (vecSrc - tr.endpos).Length();
         }
 
-        flAdjustedDamage = RemapValClamped(flDistanceToEntity, 0, flRadius, info.GetDamage(), info.GetDamage() * falloff);
-
-        // Take a little less damage from yourself
-        if (tr.m_pEnt == info.GetAttacker())
-        {
-            flAdjustedDamage = flAdjustedDamage * 0.75f;
-        }
-
-        if (flAdjustedDamage <= 0)
-            continue;
-
-        // the explosion can 'see' this entity, so hurt them!
-        if (tr.startsolid)
-        {
-            // if we're stuck inside them, fixup the position and distance
-            tr.endpos = vecSrc;
-            tr.fraction = 0.0f;
-        }
+        flAdjustedDamage = info.GetDamage() * ( 1 - ( 1 - falloff) * clamp(flDistanceToEntity / flRadius, 0, 1));
```

Comment:
You may want to use float values here, I'm not sure if clamp will be doing any funky generics stuff without it:
```suggestion
        flAdjustedDamage = info.GetDamage() * ( 1.0f - ( 1.0f - falloff) * clamp(flDistanceToEntity / flRadius, 0.0f, 1.0f));
```

Reply:
Changed to floats